mitiq/cdr/clifford_training_data.py

```diff
@@ -410,7 +410,10 @@ def _probabilistic_angle_to_clifford(
 
 
 def _is_clifford(op: cirq.ops.Operation) -> bool:
-    if isinstance(op.gate, cirq.ops.XPowGate):
+    if (
+        isinstance(op.gate, cirq.ops.XPowGate)
+        and op.gate.exponent % 2 in _CLIFFORD_EXPONENTS
+    ):
```

Comment:
@L-P-B I was thinking to have the body of this function just be `return cirq.has_stabilizer_effect(op)` (at least for single-qubit gates). Thoughts?